mainnet/jobs.json

```diff
@@ -1,202 +1,218 @@
-  }
-]
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_gemini",
+      "selector": "last",
+      "url": "https://api.gemini.com/v1/pubticker/ethusd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_kucoin",
+      "selector": "data.ETH",
+      "url": "https://api.kucoin.com/api/v1/prices?base=USD&currencies=ETH"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_kraken",
+      "selector": "result.XETHZUSD.c[0]",
+      "url": "https://api.kraken.com/0/public/Ticker?pair=ETHUSD"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_coinbase",
+      "selector": "data.amount",
+      "url": "https://api.coinbase.com/v2/prices/ETH-USD/spot"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_coingecko_api",
+      "selector": "ethereum.usd",
+      "url": "https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_bitfinex",
+      "selector": "last_price",
+      "url": "https://api.bitfinex.com/v1/pubticker/ethusd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_cryptocompare",
+      "selector": "[USD]",
+      "url": "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "ethusd_razor",
+      "selector": "ETHUSD",
+      "url": "https://datafeeds.razorscan.io/prices"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "btcusd_gemini",
+      "selector": "last",
+      "url": "https://api.gemini.com/v1/pubticker/btcusd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "btcusd_kucoin",
+      "selector": "data.BTC",
+      "url": "https://api.kucoin.com/api/v1/prices?base=USD&currencies=BTC"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "btcusd_kraken",
+      "selector": "result.XXBTZUSD.c[0]",
+      "url": "https://api.kraken.com/0/public/Ticker?pair=BTCUSD"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "btcusd_coinbase",
+      "selector": "data.amount",
+      "url": "https://api.coinbase.com/v2/prices/BTC-USD/spot"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "btcusd_coingecko_api",
+      "selector": "bitcoin.usd",
+      "url": "https://api.coingecko.com/api/v3/simple/price?ids=bitcoin&vs_currencies=usd"
+    },
+    {
+      "weight": 1,
+      "power": 2,
+      "selectorType": 0,
+      "name": "btcusd_cryptocompare",
+      "selector": "[USD]",
```

Comment:
Here as well, the selector should be `USD` instead of `[USD]` for cryptocompare API's.